cannon/mipsevm/multithreaded/instrumented.go

```diff
@@ -46,6 +48,7 @@ func (m *InstrumentedState) InitDebug() error {
 		return err
 	}
 	m.stackTracker = stackTracker
+	m.statsTracker = NewStatsTracker()
```

Comment:
`InitDebug` is only called when cannon is run with `--debug`, which requires a meta.json to work (see https://github.com/ethereum-optimism/optimism/blob/67dd69338fa417061f9ae531e369d0261f87e669/cannon/cmd/run.go#L382C1-L390C3). To avoid requiring a meta.json for metrics tracking, I suggest using a new separate function for setting up `statsTracker`. Then `statsTracker` can be initialized only if the user specifies a `--debug-info` so it's zero-overhead for the default case.

Reply:
Thanks for pointing this out!  In my head I was thinking we only had the `--debug-info` flag and that would invoke this.  Added a new `EnableStats()` method that we call if `--debug-info` is set.